model/src/main/java/org/projectnessie/api/params/CommitLogParams.java

```diff
@@ -74,6 +74,10 @@
   @QueryParam("before")
   private Instant before;
 
+  @Parameter(description = "A Common Expression Language (CEL) expression")
```

Comment:
This should prob be more descriptive. Somewhere in the docs or OpenAPI spec there should be more detail and this should link to it.

Reply:
totally agreeing with you. I just didn't have time yet to add docs

Reply:
updated the docs with some examples on the CLI & on the REST API side